pom.xml

```diff
@@ -469,7 +469,7 @@
             <dependency>
                 <groupId>com.google.cloud.bigdataoss</groupId>
                 <artifactId>gcs-connector</artifactId>
-                <version>3.1.5</version>
+                <version>3.1.6</version>
```

Comment:
Why do we update this dependency in "Update AWS SDK v2 to 2.33.5" commit?

Reply:
my bad :)

Reply:
Fixed